1699390074115	Sync.LogManager	DEBUG	Flushing file log
1699390074143	Sync.LogManager	DEBUG	Log cleanup threshold time: 1698526074143
1699390074146	Sync.LogManager	DEBUG	Done deleting files.
1699393678551	Sync.Service	DEBUG	User-Agent: Firefox/119.0 (Linux x86_64) FxSync/1.121.0.20231023160744.desktop
1699393678552	Sync.Service	INFO	Starting sync at 2023-11-07 15:47:58 in browser session PUt4QYkuCdRt
1699393678552	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1699393678554	Sync.Status	INFO	Resetting Status.
1699393678554	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1699393678555	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1699393678555	FirefoxAccounts	DEBUG	getOAuthToken enter
1699393678555	FirefoxAccounts	TRACE	getCachedToken returning cached token
1699393678555	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1699393678556	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1699393678556	Sync.SyncAuthManager	INFO	Getting sync key
1699393678557	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1699393678557	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1699393678557	FirefoxAccounts	DEBUG	getOAuthToken enter
1699393678557	FirefoxAccounts	TRACE	getCachedToken returning cached token
1699393678557	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1699393678557	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1699393678557	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1699393678561	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1699393678561	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 479\"  data: no]"}
1699393678563	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1699393678563	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1699393678563	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1699393678563	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1699393678564	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1699393678565	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1699393678568	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/147255507/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1699393678569	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1699393678569	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1699393678569	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1699393678569	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1699393678569	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1699393678572	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
